### ISSUE TITLE
Broken link to pretrained softmax model

### ISSUE THREAD

**Issue by mattiasbax** [hustvl/SparseInst#74]
Hello!

As mentioned in the title the link to download the new pretrained model using softmax is broken.
Does anyone have a working link they could share?

wbr

**Comment by wondervictor** [comment]
Hi @mattiasbax, thank you for reminding me. I'll fix it soon.

**Comment by wondervictor** [comment]
Hi @mattiasbax, we have updated the links to the models. Thanks for you interest in SparseInst. Hope it will help you.

**Comment by wondervictor** [comment]
This issue will be closed, you can open a new issue if you have other problems or reopen it. Thanks for your interest in SparseInst 😊 . If you find it useful in your work or research, could you give us a star 🌟 or help us recommend SparseInst to your friends.
